rhel7/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ ssg_build_html_nistrefs_table(${PRODUCT} "standard")
 ssg_build_html_nistrefs_table(${PRODUCT} "ospp")
 ssg_build_html_nistrefs_table(${PRODUCT} "C2S")
 ssg_build_html_nistrefs_table(${PRODUCT} "stig")
+ssg_build_html_nistrefs_table(${PRODUCT} "ccc")
```

Comment:
Probably `ospp` references would be more useful.
Right now, there is no version of `ssg_build_html_nistrefs_table` for `ospp` refs.

An alternative is to generate a product table for `ospp` references:
`ssg_build_html_table_by_ref(${PRODUCT} "ospp")`
 Although the table is not per profile, but for whole datastream, I think it will be more useful.

Reply:
There is already the same line few lines above my addition. Should I therefore just remove this addition?

Reply:
What I suggested is different actually: 
`ssg_build_html_nistrefs_table` vs
`ssg_build_html_table_by_ref`